text/rfcs/0006-metadata-in-forc-manifest.md

```diff
@@ -0,0 +1,96 @@
+- Feature Name: Metadata Section in Forc Manifest
+- Start Date: 2024-01-25
+- Author(s): @deekerno
+- RFC PR: [FuelLabs/rfcs#21](https://github.com/FuelLabs/rfcs/pull/21)
+
+## Summary
+
+An optional section describing metadata should be added to the Forc manifest.
+
+## Motivation
+
+The objective of this change is to allow a Sway contract developer to describe arbitrary information that will be available for Forc plugins to process. The immediate use of this change will be to allow for Sway contract developers to specify information for indexing the contract in the Forc manifest, if desired. The expected outcome is that developers will be able to define indexing-specific _metadata_ in the same place and manner that they define other important information, e.g. authoring information, dependencies, etc.
+
+_Note: This document will describe the change in the context of a indexer-specific example._
+
+## Guide-Level Explanation
+
+_Indexing-specific metadata_ refers to the set of information necessary to allow for an indexing service to begin processing resultant data from contract execution.The service should use this information to create distinct and separate tables for contract data as well as route creation on a publicly-available querying API.
```

Comment:
```suggestion
_Indexing-specific metadata_ refers to the set of information necessary to allow for an indexing service to begin processing resultant data from contract execution. The service should use this information to create distinct and separate tables for contract data as well as route creation on a publicly-available querying API.
```

Reply:
Just added a space after `.`, the diff seems huge for some reason.

Reply:
It's because I wrote it in Markdown with every paragraph as a line; thank word-wrapping for that one. 😅